.schema.env

```diff
@@ -35,6 +35,7 @@ EGO_SCORE_PROXY_CLIENT_SECRET=
 
 # Program Service
 PROGRAM_SERVICE_ROOT=
```

Comment:
Need to document port-forwarding requirements

Reply:
Port forward both HTTP + GRPC endpoints from selected Env
Set env vars as:
`PROGRAM_SERVICE_ROOT=localhost:50051
PROGRAM_SERVICE_HTTP_ROOT=localhost:8083`
where 50051/8083 are the standard ports, substitute w/ actual values

Reply:
Should be no port forwarding necessary when running the service from docker-compose